packages/mui-material-next/src/Switch/Switch.d.ts

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { SxProps } from '@mui/system';
+import { OverridableStringUnion } from '@mui/types';
+import { InternalStandardProps as StandardProps, Theme } from '@mui/material';
+// eslint-disable-next-line no-restricted-imports
+import type { SwitchBaseProps } from '@mui/material/internal/SwitchBase';
+import { SwitchClasses } from './switchClasses';
+
+export interface SwitchPropsSizeOverrides {}
+
+export interface SwitchPropsColorOverrides {}
+
+type NewType = SwitchBaseProps;
```

Comment:
```suggestion
```

Why was this type introduced?

Reply:
@mnajdova I have no idea myself. I think I accidentally came across a quick fix and VSCode changed it. I was wondering myself. I have fixed it.